api/src/test/java/io/strimzi/test/StrimziExtension.java

```diff
@@ -92,11 +99,14 @@
     private static final String DEFAULT_TAG = "";
     private static final String TAG_LIST_NAME = "junitTags";
     private static final String START_TIME = "start time";
+    private static final String TEST_LOG_DIR = System.getenv().getOrDefault("TEST_LOG_DIR", "/tmp/strimzi");
```

Comment:
From my POV the default path to logs should be part of system tests output directory (_./systemtest/target/..._)

Reply:
Right! This really makes sense. If we will have logs in the target, we will have them in Jenkins's UI.